src/invisible/explainers/enrichments/reference-image-mapper.md

```diff
@@ -0,0 +1,211 @@
+  </div>
+</div>
+
+
+### 2. Basketball
+
+<div class="mcontainer">
+  <div class="col-mcontainer">
+    <h4>Reference image</h4>
+    <v-img
+    class="rounded" 
+    style="margin-bottom:14px;"
+    max-width="100%" max-height="300px" contain 
+    :src="require('../../../media/invisible/rim/basketball-img.jpg')"
+  ></v-img>
+    Here we can take a photo of the basketball hoop and court background.
+  </div>
+  <div class="col-mcontainer">
+    <h4>Scanning video</h4>
+    <Youtube src="4MB5o4W_XJo"/>
+    The scanning recording, in this case, is about 45 s long – note that the angles and distances cover what a player might see when dribbling towards the basket or taking a shot
+  </div>
+</div>
+
+### 3. Supermarket shelf
+
+<div class="mcontainer">
+  <div class="col-mcontainer">
+    <h4>Reference image</h4>
+    <v-img
+    class="rounded" 
+    style="margin-bottom:14px;"
+    max-width="100%" max-height="300px" contain 
+    :src="require('../../../media/invisible/rim/supermarket-img.jpg')"
+  ></v-img>
+    This photo captures the assortment of packagíng in the coffee aisle of a supermarket.
+  </div>
+  <div class="col-mcontainer">
+    <h4>Scanning video</h4>
+    <Youtube src="d3Yk3nKDIOQ"/>
+    Notice that the scanning recording is fairly slow to reduce motion blur for this feature rich shelf.
+  </div>
+</div>
+
+### 4. An entire building
+
+<div class="mcontainer" display="grid | inline-grid">
+  <div class="col-mcontainer">
+    <h4>Reference image</h4>
+    <v-img
+    class="rounded" 
+    style="margin-bottom:14px;"
+    max-width="100%" max-height="300px" contain 
+    :src="require('../../../media/invisible/rim/building-img.jpg')"
+    ></v-img>
+    This is a photo of the <i>entire</i> building
+  </div>
+  <div class="col-mcontainer">
+    <h4>Scanning video</h4>
+    <Youtube src="0U4H-uOIHlw"/>
+    We take a longer scanning recording, about 2 min. The angles and distances cover what a person might see whilst walking past or standing in front of the building.
+  </div>
+</div>
+
+## Scanning best practices
+
+- Make the recording while holding the Pupil Invisible glasses in your hand rather than wearing it on your head.
+- Record the object of interest from all possible angles and from all distances a subject may look. More diversity is better. Collecting sufficiently diverse viewpoints or a large object, like a building, may take longer than capturing a small object like a magazine.
+- Move the glasses slowly while recording to avoid motion blur.
+- Make sure to have good contrast and that your scene lighting during scanning is similar to that during mapping.
+
+What works and what doesn’t?
+- These examples are **feature rich**. Reference Image Mapper needs enough salient visual content to produce a successful mapping.
+- The scenes have relatively **static** features in the environment. If there is a lot of movement or the objects change in appearance, the mapping *can* fail.
```

Comment:
```suggestion
- The scene needs to have relatively **static** features in the environment. If there is a lot of movement or the objects change in appearance, the mapping *can* fail.
```
Maybe this point needs some examples to illustrate? Not sure if people would understand what "objects changing in their appearance" would mean in practice 🤔